migrate/live-migration/live-migration-from-postgres.md

```diff
@@ -0,0 +1,252 @@
+---
+title: Migrate from PostgreSQL using live migration
+excerpt: Migrate from a PostgreSQL database using the low-downtime live migration method
+products: [cloud]
+keywords: [migration, low-downtime]
+tags: [migration, logical backup, replication]
+---
+
+import GettingHelp from "versionContent/_partials/_migrate_dual_write_backfill_getting_help.mdx";
+import SourceTargetNote from "versionContent/_partials/_migrate_source_target_note.mdx";
+import StepOne from "versionContent/_partials/_migrate_dual_write_step1.mdx";
+
+# Live migration from PostgreSQL database with pgcopydb
+
+This document provides detailed step-by-step instructions to migrate data using
+[pgcopydb][pgcopydb] to perform a live migration from a source database which
+is using PostgreSQL to Timescale.
+
+Before beginning the migration process, ensure that the following tools are
+installed and available on the system that performs the migration:
+
+- `psql`, `pg_dump`, and `pg_dumpall`: These can be installed by following the
+  instructions in the [How to Install psql on Mac, Ubuntu, Debian, Windows][install-psql]
+  blog post. Although the instructions specifically mention `psql`, following
+  them also installs `pg_dump` and `pg_dumpall`.
+
+- `pgcopydb`: Installation instructions can be found in the
+  [official repository][install-pgcopydb].
```

Comment:
I would prefer it if we don't send people away from our docs page for them to install our tools. Could we perhaps provide installation instructions for the one/two most common distros, and provide links for people using other distros?

It looks like we should be able to provide one set of instructions for ubuntu and debian, so I would suggest that we do that. I briefly tried to install postgres on rocky linux, and it failed with a gpg signature error, so I don't think that's worth investigating.

Reply:
Also: apparently pgcopydb isn't available for RHEL-based distros, which backs the idea of providing one set of instructions for debian/ubuntu and leaving the RHEL-strugglers to struggle.